packages/js/components/changelog/add-41048

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
Should we potentially change this to `major`, given we are changing the use of `li` and `lo` to `div`? As this is a bigger change in the output.
Also the removal of the wrapper, which devs may of relied upon ( like in our Downloads list ).